open-craft/edx-platform
# [FAL-2642] Update arabic translations for AFS.

Updates three translation strings.

**Test instructions**:

This patch was applied to the https://mtyaka.opencraft.hosting/ instance, so you can verify that the new translations show up by:

1. Creating an account on https://mtyaka.opencraft.hosting/
2. Changing the language to Arabic via https://mtyaka.opencraft.hosting/update_lang
3. Enroll into the Demo course and verify that you can see the new translation on the "View Course" button on https://mtyaka.opencraft.hosting/dashboard
4. Go to a a unit that includes the discussion widget, for example [this one](https://mtyaka.opencraft.hosting/courses/course-v1:edX+DemoX+Demo_Course/courseware/graded_interactions/simulations/?activate_block_id=block-v1%3AedX%2BDemoX%2BDemo_Course%2Btype%40sequential%2Bblock%40simulations) and verify that you can see the new translation for the "Add a Post" link/button.
5. Go to [the forum](https://mtyaka.opencraft.hosting/courses/course-v1:edX+DemoX+Demo_Course/discussion/forum/), click the "Add Post" button and verify that the "Save" button contains the new translation.

**Reviewers**
